addons/addon-base-raas/packages/base-raas-services/lib/schema/create-aws-accounts.json

```diff
@@ -39,6 +39,14 @@
     "encryptionKeyArn": {
       "type": "string",
       "pattern": "^arn:aws:kms:.*$"
+    },
+    "appStreamStackName": {
+      "type": "string",
+      "minLength": 10
+    },
+    "appStreamSecurityGroup": {
```

Comment:
This is an ID, how about we name it like: `appStreamSecurityGroupId`

Reply:
updated